data/formats-data.js

```diff
@@ -944,7 +944,7 @@ exports.BattleFormatsData = {
 	},
 	lickilicky: {
 		randomBattleMoves: ["wish","protect","bodyslam","knockoff","dragontail","healbell","swordsdance","explosion","earthquake","powerwhip"],
-		randomDoubleBattleMoves: ["wish","protect","dragontail","knockoff","bodyslam","return","powerwhip","swordsdance","earthquake","toxic","healbell","explosion"],
+		randomDoubleBattleMoves: ["wish","protect","dragontail","knockoff","bodyslam","rockslide","powerwhip","earthquake","toxic","healbell","explosion"],
```

Comment:
Yes. You can't and won't use Protect/Wide Guard every turn, and it can catch people off guard.

> [14:07]	**Ascriptmaster**: !usage lickilicky moves doubles
> [14:07]	**TIBot**: Explosion 42.055% (it was Lickilicky's most common move)